### PR TITLE
Stress test local finished

### PR DESCRIPTION
**Pull request by michaelrtobias**

### CODE REVIEW

**Comment by michaelrtobias** [comment]
@bland0 @chanceknguyen

File: db/postgres/schema.sql [hunk 1]
```diff
@@ -26,7 +26,7 @@ CREATE TABLE games (
 
 -- games table
 COPY games (id, titleCover, title, price, aboutInfo, requirements, genre, developer, publisher, releaseDate, steamAcheivments, languages, attributes, moreLikeThis)
-FROM '/Users/michael/Documents/Galvanize/HRR48/SDC/SDC-Project/body/gamesPostgreSQLTest.csv'
+FROM '/Users/michael/Documents/Galvanize/HRR48/SDC/SDC-Project/body/games.csv'
```
Comment (bland0) [inline db/postgres/schema.sql:29]: This is a path specific to your machine.

**Review by bland0 — COMMENTED** [review]
Only one issue with that local path.

File: db/postgres/schema.sql (hunk 1 above)
Comment (michaelrtobias) [reply db/postgres/schema.sql:29]: fixing now

**Pull request merged**